src/compiler/checker.ts

```diff
@@ -19372,7 +19361,17 @@ namespace ts {
             }
         }
 
+        function checkJsDoc(node: FunctionDeclaration | MethodDeclaration) {

```

Comment:
I'd like to keep the function around for readability and because it may someday be used from more locations.